src/main/webapp/view/transcript/transcript.js

```diff
@@ -171,3 +173,43 @@ function scrollToTopOfTranscript(transcriptLine) {
   const ulElementOffset = transcriptLine.parentElement.offsetTop;
   transcriptContainer.scrollTop = transcriptLine.offsetTop - ulElementOffset;
 }
+
```

Comment:
All of this transcript seeking code should have been made into its own class and moved out of this file.

Reply:
I agree! It is currently being worked on in another pull request #262. I added TODOs on that pull request to move these functions over once it is merged.

Reply:
nit: Where was the TODO added? I do not see it. It is better to add the TODO right above the method (under the JSDoc) so when people are reviewing they see the TODO.